File: services/identity-service/src/main/java/br/com/deltaglobalbank/identity/features/assignUserRole/AssignRolesResponse.java
package br.com.deltaglobalbank.identity.features.assignUserRole;

import java.util.List;
import java.util.UUID;

import br.com.deltaglobalbank.identity.domain.role.RoleCode;

public record AssignRolesResponse(
    UUID userId,
    List<RoleCode> addedRoles,
    List<RoleCode> currentRoles
) {
}

File: services/identity-service/src/main/kotlin/br/com/deltaglobalbank/identity/features/assignUserRole/AssignRolesResponse.kt
package br.com.deltaglobalbank.identity.features.assignUserRole

import br.com.deltaglobalbank.identity.domain.role.RoleCode
import java.util.UUID

data class AssignRolesResponse(
    val userId : UUID,
    val addedRoles : List<RoleCode>,
    val currentRoles : List<RoleCode>
) {
}